src/orca-loader/Elements.res

```diff
@@ -28,6 +28,7 @@ let make = (
     let savedPaymentElement = Dict.make()
     let localOptions = options->JSON.Decode.object->Option.getOr(Dict.make())
     let endpoint = ApiEndpoint.getApiEndPoint(~publishableKey, ())
+    let redirect = ref("if_required")
```

Comment:
There is no need to have redirect as a ref

Reply:
It's required to set the value in that context and use this.